src/main/java/athena/SortByStartTime.java

```diff
@@ -0,0 +1,18 @@
+package athena;
+
+import athena.task.Task;
+
+import java.util.Comparator;
+
+public class SortByStartTime implements Comparator<Task> {
```

Comment:
Since this is a class/object I think it would be better to name it with a noun. Something like `TaskTimeComparator`? I think that this also helps us immediately tell that it is a comparator when looking at the name.